.travis.yml

```diff
@@ -12,63 +12,73 @@ php:
 # Set up environment variables.
 env:
   global:
-    # add composer's global bin directory to the path
-    # see: https://github.com/drush-ops/drush#install---composer
+    # Global
     - PATH="$PATH:$HOME/.composer/vendor/bin"
 
-    # Configuration variables.
+    # Composer
+    - COMPOSER_EXIT_ON_PATCH_FAILURE=1
+
+    # DRUPAL_TI Config.
     - DRUPAL_TI_MODULE_NAME="contacts"
-    #- DRUPAL_TI_SIMPLETEST_GROUP="user"
-    - DRUPAL_TI_PULL_SIMPLETEST_GROUP="contacts"
-    - DRUPAL_TI_NON_PULL_SIMPLETEST_GROUP="contacts"
-    #- DRUPAL_TI_NON_PULL_SIMPLETEST_GROUP="user,contacts"
-    #- DRUPAL_TI_NON_PULL_SIMPLETEST_GROUP="--all"
+    - DRUPAL_TI_SIMPLETEST_GROUP="contacts"
 
-    # The environment to use, supported are: drupal-7, drupal-8
+    # DRUPAL_TI Env
     - DRUPAL_TI_ENVIRONMENT="drupal-8"
     - DRUPAL_TI_CORE_BRANCH="8.2.x"
 
-    # Drupal specific variables.
+
+    # Define runners and environment vars to include before and after the
+    # main runners / environment vars.
+    - DRUPAL_TI_SCRIPT_DIR_BEFORE="./travis-ci/before"
+    - DRUPAL_TI_SCRIPT_DIR_AFTER="./travis-ci/after"
+
+    # DRUPAL_TI Testing.
     - DRUPAL_TI_DB="drupal_travis_db"
     - DRUPAL_TI_DB_URL="mysql://root:@127.0.0.1/drupal_travis_db"
-    # Note: Do not add a trailing slash here.
     - DRUPAL_TI_WEBSERVER_URL="http://127.0.0.1"
     - DRUPAL_TI_WEBSERVER_PORT="8080"
-
-    # Simpletest specific commandline arguments, the DRUPAL_TI_SIMPLETEST_GROUP is appended at the end.
     - DRUPAL_TI_SIMPLETEST_ARGS="--verbose --color --concurrency 4 --url $DRUPAL_TI_WEBSERVER_URL:$DRUPAL_TI_WEBSERVER_PORT"
+    - DRUPAL_TI_PHPUNIT_ARGS="--group contacts"
```

Comment:
Discussed with @yanniboi, we will (as a follow up) switch to using `BrowserTestBase` and drop running simpletest.